frontend/www/js/omegaup/course/new.ts

```diff
@@ -43,22 +44,24 @@ OmegaUp.on('ready', function () {
                 accept(null);
               }
             })
-              .then(function (school_id) {
-                const params = {
+              .then((schoolId: any) => {
```

Comment:
hay que cambiar L34 por

```typescript
            new Promise(<number|null>(accept, reject) => {
```

así que esta línea debería ser 

```suggestion
              .then((schoolId) => {
```

porque el checador de tipos puede inferir que el único tipo válido para `schoolId` es `number|null` debido al cambio de la línea de arriba.

Reply:
Ya hice el cambio, pero me sigue mandando el mismo error

Reply:
seguro que es el mismo? https://github.com/omegaup/omegaup/pull/4383/checks?check_run_id=873795095#step:4:20 dice 

```
  ERROR in /home/runner/work/omegaup/omegaup/frontend/www/js/omegaup/course/new.ts(58,19):
  58:19 Type 'number | null' is not assignable to type 'number | undefined'.
    Type 'null' is not assignable to type 'number | undefined'.
      56 |                   admission_mode: omegaup.AdmissionMode.Private,
      57 |                   assignments: [],
    > 58 |                   school_id: schoolId,
         |                   ^
      59 |                 };
      60 | 
      61 |                 if (source.unlimitedDuration) {
```

tons faltaría cambiar L58 por

```typescript
                   school_id: schoolId ?? undefined,
```

Reply:
Listo